modules/bundles/get_started.ipynb

```diff
@@ -164,8 +164,6 @@
     "        \"$import os\",\n",
```

Comment:
<p>I would have said the metadata file was required since it would include information we'd always want to have. </p><p> </p><p><code></code><code></p><p>Optionally, we can define a <code>metadata</code> file in the bundle, which contains the metadata information relating to the model, including what the shape and format of inputs and outputs are, what the meaning of the outputs are, what type of model is present, and other information. The structure is a dictionary containing a defined set of keys with additional user-specified keys.</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Sure, updated in the latest commit.

Thanks.